### PR TITLE
Fixes for lean draft

### PR DESCRIPTION
**Pull request by danjoa**
→ we need these for testing SFlight with lean draft

### CODE REVIEW

**Review by David-Kunz — APPROVED** [review]
Not sure if those defined users (`martha` etc.) are overwritten by `mocked-auth`

File: app/travel_processor/labels.cds [hunk 1]
```diff
@@ -6,7 +6,7 @@ using { sap.fe.cap.travel as schema } from '../../db/schema';
 
 annotate schema.Travel with @title: '{i18n>Travel}' {
   TravelUUID   @UI.Hidden;
-  TravelID     @title: '{i18n>TravelID}'      @Common.Text: Description;
+  TravelID     @title: '{i18n>TravelID}';
```
Comment (danjoa) [inline app/travel_processor/labels.cds:9]:
```suggestion
  TravelID @title: '{i18n>TravelID}' @Common.Text: Description; // remove common text for one-liner displays
```
revert

File: app/travel_processor/layouts.cds [hunk 2]
```diff
@@ -45,6 +45,10 @@ annotate TravelService.Travel with @UI : {
       Value : TravelID,
       ![@UI.Importance] : #High
     },
+    {
+      Value : Description,
+      ![@UI.Importance] : #High
+    },
```
Comment (danjoa) [inline app/travel_processor/layouts.cds:51]:
```suggestion
// Add for one-liner displays
//    { 
//      Value : Description,
//      ![@UI.Importance] : #High
//    },
```

File: package.json [hunk 3]
```diff
@@ -46,20 +46,22 @@
...
                 "reviewer"
               ]
             },
             "amy": {
-              "password": "amy",
+              "roles": [
+                "processor"
+              ]
+            },
+            "me": {
```
Comment (danjoa) [inline package.json:64]:
```suggestion
```

File: package.json [hunk 4]
```diff
@@ -46,20 +46,17 @@
...
             },
             "martha": {
-              "password": "martha",
               "roles": [
                 "reviewer"
               ]
             },
             "amy": {
-              "password": "amy",
               "roles": [
```
Comment (danjoa) [inline package.json:60]:
```suggestion
              "password": "amy",
              "roles": [
```

**Pull request closed without merging**